DXRVanilla/DeusEx/Classes/PersonaScreenInventory.uc

```diff
@@ -179,7 +179,7 @@ function EnableButtons()
         inv = Inventory(selectedItem.GetClientObject());
 
         if (inv != None) {
-            if (class'DXRLoadouts'.static.IsRefused(inv.class)) {
+            if (class'DXRLoadouts'.static.GetLootAction(inv.class) == 1) {
```

Comment:
interesting to consider what should happen for items set to autoconsume (action == 2), maybe there should be no button for Trash/Untrash? since there's no way to put it back to autoconsume, and they don't vary by seed anyways

Reply:
I think this is the only consideration I have left for this PR, everything else looks good

so maybe either hide this button or disable it when action == 2